lib/install.ps1

```diff
@@ -739,6 +739,7 @@ function link_current($versiondir) {
     }
 
     cmd /c mklink /j $currentdir $versiondir | out-null
+    attrib $currentdir +R /L
```

Comment:
I think this is causing an error while uninstalling an app
![image](https://cloud.githubusercontent.com/assets/442683/24432151/71ba9d50-1420-11e7-82bd-685a8e6a0719.png)

"Zugriff verweigert" is german for "Permission denied" ;-)